kingpin/actors/rightscale/deployment.py

```diff
@@ -119,13 +116,119 @@ def _execute(self):
             self._client._client.deployments, params)
 
 
+class Clone(DeploymentBaseActor):
+
+    """Clones a RightScale deployment.
+
+    **Options**
+
+    :clone:
+      The name of the deployment to clone.
+
+    :name:
+      The name of the deployment to create.
+
+    :description:
+       The description of the new deployment.
+       (*optional*)
+
+    :server_tag_scope:
+       The routing scope for tags for servers in the deployment.
+       Can be 'deployment' or 'account'
+       (*optional*, default: deployment)
+    """
+
+    all_options = {
+        'clone': (str, REQUIRED, 'The name of the deployment to clone.'),
```

Comment:
I wanted to keep `name` because that's the RS API name for the destination name.  http://reference.rightscale.com/api1.5/resources/ResourceDeployments.html ... thoughts?